actions/README.md

```diff
@@ -35,6 +35,20 @@ will expand into.
 |--------------|--------------------------------------------------------|
 | `${version}` | The version of the release (i.e., the name of the tag) |
 
+## DatadogReleaseMetric
+
+A GitHub action to submit a Release metric to Datadog.
+
+### Usage
+
+```yml
+      - name: Submit metric to Datadog
+        uses: sendgrid/dx-automator/actions/datadog-release-metric@main
+        env:
+          DD_API_KEY: ${{ secrets.DD_API_KEY }}
```

Comment:
Note that the secret name is actually `DATADOG_API_KEY` in this repo's secrets.

Reply:
Was that changed recently? I think it used to be DD_API_KEY, since that's what worked in the test workflow

Reply:
Yeah, that was my bad. I changed it to `DATADOG_API_KEY` since that's what I used here: https://github.com/sendgrid/dx-automator/blob/ade01eeb95c503eacf90c050605ded2bef8371d9/.github/workflows/sonar_cloud_metrics.yml#L27

👍  to making it consistent. Either is fine; think I prefer `DD_API_KEY` to be less confusing.